.github/pull_request_template.md

```diff
@@ -1,22 +1,27 @@
-#### Link this PR to an issue
-Fixes #ISSUE-NUMBER
+### Link this PR to an issue
 
-#### Type of change (Remove other not matching type)
+_#ISSUE-NUMBER_
 
-- Bug fix (non-breaking change which fixes an issue)
-- New feature (non-breaking change which adds functionality)
-- Breaking change (fix or feature that would cause existing functionality to not work as expected)
-- Documentation update
+### Type of change (Remove other not matching type)
 
-#### Describe the changes you have made in this PR -
+- `fix`: Bug fix (non-breaking change which fixes an issue)
+- `feat`: New feature (non-breaking change which adds functionality)
+- `feat!`: Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- `docs`: Documentation update
 
-#### Screenshots of the changes (If any) -
+### Describe the changes you have made in this PR
```

Comment:
what do you think about moving this higher - above type of change? 
I found this more relevant so far

Reply:
👍 means we move the "description" up? :)

Reply:
Haha , wait. So current order is:
- Link issue
- Type of change
- Description
- Screenshot
- Tested
- Checklist

Ist that ok?

Reply:
I thought about: 


1. Description
2. Link issue
3. Type of change
4. Screenshot
5. Tested
6. Checklist

Reply:
Done